Makefile

```diff
@@ -43,7 +47,7 @@ $(DEV_STAMP): $(PYTHON) dev-requirements.txt
 
 virtualenv: $(PYTHON)
 $(PYTHON):
-	$(VIRTUALENV) $(VENV)
+	$(VIRTUALENV) -p pypy $(VENV)
```

Comment:
Is there any risk that `pypy` might point to a python3 version of pypy?

Reply:
Not really, otherwise it'd break a lot of stuff. They've pretty closely tied pypy to python2.7

Reply:
I'm not really sure that we should allow a given system's python2 implementation. Pypy has openly stated that [they're going to support a version of the Python2 interface](https://doc.pypy.org/en/latest/faq.html#how-long-will-pypy-support-python2), so we can presume that they'll address critical fixes, rather than some potential abandonware install.

Reply:
That's fine by me 👍